ppr-ui/src/components/dialogs/SecuredPartyDialog.vue

```diff
@@ -259,7 +265,8 @@ export default defineComponent({
         localState.party.action = ActionTypes.EDITED
         setRegisteringParty(localState.party)
       } else {
-        addSecuredParty(localState.party, props.activeIndex)
+        currentSecuredParty.value = localState.party
```

Comment:
Keep in mind, in javascript this is only a reference to the original object (localstate.party), since thats how js works with values that are not primitive. 
If that is your intention, all good, but any changes made to party would also reflect in currentSecuredParty. 

This is a blessing and curse in JS. Just keep in mind, if you wanted to decouple, you would need to make a copy of the object, not reference it 👍

Reply:
This is the intention. I needed a reference of the info on the form in useSecuredParty.ts which updates as information is input/removed from the form. Thanks for the info!